paymentsheet/src/main/java/com/stripe/android/paymentsheet/verticalmode/PaymentMethodRowButton.kt

```diff
@@ -66,7 +66,7 @@ private fun TitleContent(title: String, subtitle: String?, isEnabled: Boolean,)
             val subtitleTextColor = MaterialTheme.stripeColors.subtitle
             Text(
                 text = subtitle,
-                style = MaterialTheme.typography.subtitle1.copy(fontSize = 10.sp),
+                style = MaterialTheme.typography.subtitle1.copy(fontSize = 12.sp),
```

Comment:
```suggestion
                style = MaterialTheme.typography.caption,
```

Reply:
Caption results in a bolder font, which isn't what we want.

Reply:
Hmm, seems like a gap in the design system. Let’s revisit it while merging this for now.